unit_tests/ngp_algorithms/UnitTestEnthalpyDiffFluxCoeffAlg.C

```diff
@@ -0,0 +1,49 @@
+/*------------------------------------------------------------------------*/
+/*  Copyright 2019 National Renewable Energy Laboratory.                  */
+/*  This software is released under the license detailed                  */
+/*  in the file, LICENSE, which is located in the top-level Nalu          */
+/*  directory structure                                                   */
+/*------------------------------------------------------------------------*/
+
+#include "kernels/UnitTestKernelUtils.h"
+#include "UnitTestHelperObjects.h"
+
+#include "ngp_algorithms/EnthalpyEffDiffFluxCoeffAlg.h"
+
+TEST_F(EnthalpyABLKernelHex8Mesh, NGP_enthalpy_eff_diff_flux_coeff)
+{
+  // Only execute for 1 processor runs
+  if (bulk_.parallel_size() > 1) return;
+
+  LowMachKernelHex8Mesh::fill_mesh_and_init_fields();
```

Comment:
You don't need to _disambiguate_ here, I had to do that for my test because there are multiple `fill_mesh_and_init_fields` for the `KsgsKernelHex8Mesh` with default arguments.